index.bs

```diff
@@ -33,6 +33,10 @@ urlPrefix: https://w3c.github.io/sensors; spec: GENERIC-SENSOR
     text: default sensor
     text: construct a sensor object; url: construct-sensor-object
     text: sensor type
+urlPrefix: https://w3c.github.io/screen-orientation; spec: SCREEN-ORIENTATION
+  type: dfn
+    text: current orientation type
```

Comment:
`text: current orientation type, url: dfn-current-orientation-type`

(Or whatever it takes to fix the link.)